clang/test/Analysis/Checkers/WebKit/retain-ptr-ctor-adopt-use-arc.mm

```diff
@@ -97,3 +97,13 @@ void create_member_init() {
 RetainPtr<id> return_bridge_cast() {
   return bridge_cast<CFArrayRef, NSArray>(create_cf_array());
 }
+
+void mutable_copy() {
+  RetainPtr<NSMutableDictionary> mutableArray = adoptNS(@{
+    @"Content-Type": @"text/html",
+  }.mutableCopy);
+}
+
+void string_copy(NSString *str) {
+  RetainPtr<NSString> copy = adoptNS(str.copy);
+}
```

Comment:
Can we also add a test for `RetainPtr<NSMutableDictionary>`?

Reply:
There is a test case for `RetainPtr<NSMutableDictionary>` right above it. Or are you saying that we should add a test case which takes NSMutableDictionary* as a function argument?

Reply:
Sorry, I meant `RetainPtr<NSArray>`

Reply:
ah, sure, I can add one.